### DISCUSSION TITLE
Rest of Generation 7

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62242] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey, this tool is awesome! I just wanted to ask when you're adding the rest of Generation VII (the UltraSun/UltraMoon stuff) because I want it all in my fangame. No need for Gens 8 and 9, just the rest of UltraSun/UltraMoon.

**Comment by queenofcorgis** [comment]
Hi @Marioipod13,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this game. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
